aio/aio-proxy/aio_proxy/response/format_search_results.py

```diff
@@ -124,6 +120,24 @@ def get_field(field, default=None):
         if include_score:
             result_formatted["score"] = get_field("meta")["score"]
 
+        # Select fields to return
+        etablissements = format_etablissements_list(
+            get_field("etablissements"), is_non_diffusible
+        )
+        score = get_field("meta")["score"]
+        slug = get_field("slug")
+
+        minimal_response = search_params.minimal
+        include_fields = search_params.include
+        if minimal_response:
+            select_fields_to_include(include_fields, result_formatted)
+
+        include_admin_fields = search_params.include_admin
+        if include_admin_fields:
+            select_admin_fields(
+                include_admin_fields, etablissements, score, slug, result_formatted
+            )
+
```

Comment:
I find it odd to systematically declare variable you might not use 

Also `search_params.minimal` and `search_params.include_admin` are quite self explanatory and do not necesssarly need redeclaration IMO

```
        if search_params.minimal:
            select_fields_to_include(search_params.include, result_formatted)

        if search_params.include_admin:
	    etablissements = format_etablissements_list(
	            get_field("etablissements"), is_non_diffusible
	        )
	    score = get_field("meta")["score"]
	    slug = get_field("slug")

            select_admin_fields(
                include_admin_fields, etablissements, score, slug, result_formatted
            )
```